pages/docs/integrations/grouparoo-braze.mdx

```diff
@@ -24,7 +24,9 @@ Braze is a customer engagement platform. Braze allows you to leverage contextual
   </tr>
   <tr>
     <td>
-      <strong>Braze Export Users</strong>
+      <a href="#braze-export-users">
+        <strong>Braze Export Users</strong>
```

Comment:
I see some docs use bold text for this column and some use regular weight.  Does the bold text mean something?

Reply:
Good Question. That's something I have been trying to keep standard. In the overview tables it is less meaningful and maybe this is where I have been less consistent. In other tables I try to bold anything that is a field name.

Reply:
I think that I will probably stick to having the destination and source type names at regular weight. Will change this.